### DISCUSSION TITLE
With crazyswarm, Is it possible to get the LOG variables from firmware?

### DISCUSSION THREAD

**Discussion by IksuLoL** [bitcraze/discussions#520] [Q&A] [answered]
Hi,

We wrote some new modules in firmware and used the LOG and PARAM frameworks. To update the variables, which were registered by using PARAM, works fine with crazyswarm. But I didn't find the API for getting the variables from the LOG section. 

My question is, is it possible to read the logged variables with crazyswarm?

Thank you very much!

**Comment by krichardsson** [comment, marked as answer]
Hi! this issue seems to be related to Crazyswarm that we don't maintain ourselves. You would be better off putting this question on Crazyswarm's discussion board if you haven't already: https://github.com/USC-ACTLab/crazyswarm/discussions

Thanks!

**Reply by IksuLoL** [reply]
Thank you for your answer, and i will try there
